scripts/tcpperf_client.py

```diff
@@ -66,12 +66,13 @@ def r(self):
         return (s, v)
 
     def text(self):
-        return 'sensor{}|{}\n'.format(*self.r()).encode('utf8')
+        a, b = self.r()
+        return f'sensor{a}|{b}\n'.encode('utf8')
```

Comment:
This was nicer before.

Reply:
Yeah, there were some parts where I was doubting how to do it nicer. However in this particular instance I think it would be better to unpack it and f-string it. It would be much clearer what is happening when reading the code.

I do agree that my choice of variable names lacks a certain ... finesse ... :-)

In the `format`-way, you magically unpack, but would need to go looking for braces. Whereas as you store them in variables, it's clearer to the reader what is happening without using too much brain power (or looking into what `r()` is doing).

Reply:
please revert.